core/esmf-aspect-model-aas-generator/src/main/java/org/eclipse/esmf/aspectmodel/aas/AspectModelAASVisitor.java

```diff
@@ -192,7 +170,7 @@ public Environment visitBase( final ModelElement base, final Context context ) {
    public Environment visitAspect( final Aspect aspect, Context context ) {
```

Comment:
`context` should be final, method arguments should not be reassigned

Reply:
Done.